ansible/roles/util-cfg-service/templates/ui/de.properties.j2

```diff
@@ -73,6 +73,11 @@ org.iplantc.services.toolRequests                              = {{ terrain.base
 org.iplantc.services.toolRequests.statusCodes                  = {{ terrain.base }}/tool-requests/status-codes
 org.iplantc.services.admin.toolRequests                        = {{ terrain.base }}/admin/tool-requests
 
+# Permanent Id Endpoint
+org.iplantc.services.permIdRequests                              = {{ terrain.base }}/permanent-id-requests
+org.iplantc.services.permIdRequests.statusCodes                  = {{ terrain.base }}/permanent-id-requests/status-codes
+org.iplantc.services.admin.permIdRequests                        = {{ terrain.base }}/admin/permanent-id-requests
+
```

Comment:
I don't think these config names have to be so far apart from their `=` characters and values.
It's also pretty minor and doesn't require a new commit for this pull request ;)